bundles/org.openhab.binding.ondilo/src/main/java/org/openhab/binding/ondilo/internal/OndiloBridge.java

```diff
@@ -100,28 +104,32 @@ public synchronized void pollOndiloICOs() {
         try {
             OndiloApiClient apiClient = this.apiClient;
             if (apiClient != null) {
+                String userInfoJson = apiClient.get("/user/info");
+                logger.trace("userInfo: {}", userInfoJson);
+                // Parse JSON to DTO
+                Gson gson = new Gson();
+                UserInfo userInfo = gson.fromJson(userInfoJson, new TypeToken<UserInfo>() {
+                }.getType());
+                if (userInfo != null) {
+                    bridgeHandler.updateUserInfo(userInfo);
+                }
+
+                // Pause for 1 second between polls in order to keep API rate limits
+                Thread.sleep(1000);
```

Comment:
This should be the responsibility of the apiclient. The bridge should not be aware of any implementation details rate limits or protocol the api client needs to make succesfull communication.

There are more `sleep` calls, the same apply's there.

Reply:
did you push that commit, because i don't see a change.

Reply:
in the meanwhile it's pushed